src/main/isDevMode.js

```diff
@@ -0,0 +1,4 @@
+// defaultApp property is added by electron in devmode.
+// It means the app is running in the "default" electron app
+// as opposed to a purposely-built package, as in production.
+export default !!process.defaultApp;
```

Comment:
What's the double `!!` signify? I don't think I've seen that yet.

Reply:
It's a conventional way to get a pure true false out of a variable that may or may not be `undefined`. Try it out!